### PR TITLE
Added University of Idaho CS Dept.

### PR DESCRIPTION
**Pull request by jimaf**
# Contributing to CSrankings

Thanks for contributing to CSrankings! Here are some guidelines to getting your pull request accepted.

**Inclusion criteria**

- [x] Make sure that any faculty you add meet the inclusion
criteria. Eligible faculty include only full-time, tenure-track
faculty members on a given campus who can advise PhD students in
Computer Science. Faculty not in a CS department or similar who can
advise PhD students in CS can be included regardless of their home
department.

**Updating an affiliation or home page**

- [x] Update affiliations, home pages, and Google Scholar entries by modifying `csrankings.csv`. For the Google Scholar entry, just use the alphanumeric identifier in the middle of the URL. If none is there, put NOSCHOLARPAGE.

**Adding one or more faculty members (including an entire department)**

- [x] If the department is not yet listed in CSrankings, the entire faculty needs to be added (not just one faculty member).

- [x] Enter each faculty member's [DBLP](http://dblp.org) name, home page, and Google Scholar entry (just the alphanumeric identifier, not the whole URL) by modifying `csrankings.csv`; include disambiguation suffixes like 0001 as needed. If the faculty entry is currently ambiguous, please do not include them. Send mail to the DBLP maintainers (dblp@dagstuhl.de) with a few publications by a particular faculty member; also, open an issue so that when the DBLP database is updated, that faculty member's information can be added.

- [x] If DBLP has multiple entries for this person, all of them need to be listed. If an alias is not already present in `dblp-aliases.csv`, add it.

- [x] If the institution you are adding is not in the US,
update `country-info.csv`.

### CODE REVIEW

File: csrankings.csv [hunk 1]
```diff
@@ -13028,3 +13028,18 @@ Xiaodong Li,RMIT University,https://titan.csit.rmit.edu.au/~e46507/,AQewL04AAAAJ
 Xiao-Dong Li,RMIT University,https://titan.csit.rmit.edu.au/~e46507/,AQewL04AAAAJ
 Xun Yi,RMIT University,https://www.rmit.edu.au/contact/staff-contacts/academic-staff/y/yi-professor-xun,oDRazncAAAAJ
 Zahir Tari ,RMIT University,https://www.rmit.edu.au/contact/staff-contacts/academic-staff/t/tari-professor-zahir,Pz9Z7C0AAAAJ
+Jim Alves-Foss,University of Idaho,http://www2.cs.uidaho.edu/~jimaf,dOaERUAAAAAJ
+Daniel Conte de Leon,University of Idaho,http://www.cs.uidaho.edu/,aLH-yuoAAAAJ
+Michael Haney,University of Idaho,http://www.cs.uidaho.edu,yFPTY8wAAAAJ
+Robert B. Heckendorn,University of Idaho,http://marvin.cs.uidaho.edu/~heckendo/,Hd0x1I4AAAAJ
+Robert E. Hiromoto,University of Idaho,http://www.cs.uidaho.edu,NOSCHOLARPAGE
+Hasan M. Jamil,University of Idaho,http://www.cs.uidaho.edu,3x4_nY4AAAAJ
+Clinton L. Jeffery,http://www.cs.uidaho.edu,NOSCHOLARPAGE
```
Comment (emeryberger) [inline csrankings.csv]: All of the entries below this point and including this one are ill-formed (they don't include the institution name). There are also two spurious newlines. Finally, the checklist should be updated to indicate that you have read and followed the instructions. Thanks.

**Review by emeryberger — CHANGES_REQUESTED** [review]

File: csrankings.csv (hunk 1 above)
Comment (emeryberger) [reply csrankings.csv]: The raw URL http://www.cs.idaho.edu is also not suitable as a homepage entry for faculty.

**Comment by emeryberger** [comment]
Please remove the .nojekyll file.

**Comment by jimaf** [comment]
Obviously I am not doing this correctly. I branched a fork, and github tries to build from my fork – sending me an error. I want to stop seeing that error message.

I committed my changes to csrankings.csv and submitted a pull request for that. But I did not submit a pull request for the .nojekyll – thinking it would be just exist on my fork

Sigh.

-jimaf

**Review by emeryberger — APPROVED** [review]

**Comment by emeryberger** [comment]
No problem. Thanks!

**Pull request merged**